### ISSUE TITLE
Add Model data in Layout file

### ISSUE THREAD

**Issue by dumithsalinda** [HansSchouten/PHPageBuilder#70]
Is it possible to access the database from the layout , the same way which can access in the blocks.
 model.php and  config.php 

Thank you

**Comment by HansSchouten** [comment]
Yes, your layout can be a `view.php` that can include any php code to access other data sources. However at the moment it does not support a `model.php` file yet.
